llvm/test/CodeGen/AMDGPU/neg_ashr64_reduce.ll

```diff
@@ -0,0 +1,102 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdpal -mcpu=gfx900 < %s | FileCheck %s
```

Comment:
why pal?

Reply:
Because amdhsa errors on the shader calling conventions but it's useful to use them for the return in SGPRs